torch/testing/_internal/common_utils.py

```diff
@@ -2590,6 +2583,22 @@ def get_tensors_from(args, kwargs):
     return set([arg for arg in args if isinstance(arg, Tensor)] +
                [v for v in kwargs.values() if isinstance(v, Tensor)])
 
+
+def has_breakpad():
+    # We always build with breakpad in CI
+    if IS_IN_CI:
```

Comment:
Don't we want to reduce the use of this flag?
This would effectively hide any bug in the code below in CI.

Reply:
True but the flip side is that maybe breakpad gets turned off in CI somehow, we have no way of noticing due to the skipIfs